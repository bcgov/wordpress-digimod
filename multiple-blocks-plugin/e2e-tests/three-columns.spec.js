import {
    createNewPost,
    enablePageDialogAccept,
    insertBlock,
    getEditedPostContent
} from '@wordpress/e2e-test-utils';

describe( 'Three columns block', () => {
    jest.setTimeout(60000);

    beforeAll( async () => {
        await enablePageDialogAccept();
    } );
    beforeEach( async () => {
        await createNewPost();
    } );

    it( 'Three columns block should be available', async () => {
        await insertBlock( 'three-columns' );
		// Check if block was inserted
        
		expect( await page.$( '[data-type="multiple-blocks-plugin/three-columns"]' ) ).not.toBeNull();

		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );
} );