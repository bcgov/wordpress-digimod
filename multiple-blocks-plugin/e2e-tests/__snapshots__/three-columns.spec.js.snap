// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Three columns block Three columns block should be available 1`] = `
"<!-- wp:multiple-blocks-plugin/three-columns -->
<div class=\\"row\\"><!-- wp:multiple-blocks-plugin/dm-col-sm-12-md-4 -->
<div class=\\"col-sm-12 col-md-4\\"><!-- wp:paragraph -->
<p></p>
<!-- /wp:paragraph --></div>
<!-- /wp:multiple-blocks-plugin/dm-col-sm-12-md-4 -->

<!-- wp:multiple-blocks-plugin/dm-col-sm-12-md-4 -->
<div class=\\"col-sm-12 col-md-4\\"><!-- wp:paragraph -->
<p></p>
<!-- /wp:paragraph --></div>
<!-- /wp:multiple-blocks-plugin/dm-col-sm-12-md-4 -->

<!-- wp:multiple-blocks-plugin/dm-col-sm-12-md-4 -->
<div class=\\"col-sm-12 col-md-4\\"><!-- wp:paragraph -->
<p></p>
<!-- /wp:paragraph --></div>
<!-- /wp:multiple-blocks-plugin/dm-col-sm-12-md-4 --></div>
<!-- /wp:multiple-blocks-plugin/three-columns -->"
`;
